plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueExpressionUtil.java

```diff
@@ -48,6 +49,19 @@
     private static final String DISJUNCT_SEPARATOR = " OR ";
     private static final Joiner DISJUNCT_JOINER = Joiner.on(DISJUNCT_SEPARATOR);
 
+    /**
+     * AWS Glue uses internally <a href="http://jsqlparser.sourceforge.net/home.php">JSQLParser</a>
+     * as SQL statement parser for the expression that can be used to filter the partitions.
+     * JSQLParser defines a set of reserved keywords which cannot be used as column names in the filter.
+     *
+     * @see <a href="https://sourceforge.net/p/jsqlparser/code/HEAD/tree/trunk/src/main/javacc/JSqlParserCC.jj">JSqlParser Grammar</a>
```

Comment:
HEAD link will become broken if file rename.
possible to link to a snapshot, commit-ish?

Reply:
https://sourceforge.net/p/jsqlparser/code/HEAD/tree/tags/jsqlparser-0.8.0/src/main/javacc/JSqlParserCC.jj

Last known Git Tag (from year 2012)